kotsadm/pkg/socket/README.md

```diff
@@ -0,0 +1,8 @@
+# Socket
+
+We started using https://github.com/graarh/golang-socketio
```

Comment:
Should we include a license header in the copied files? https://github.com/graarh/golang-socketio/blob/master/LICENSE.MIT

Reply:
Also... Is this the library that operator is using? Would it be best to use the same lib on both sides there?

Reply:
actually, is there just a way that we could share `kotsadm/pkg/socket/` and `kotsadm/operator/pkg/socket/`?

Reply:
i think that's a good idea

Reply:
it is the same library that the operator uses, yeah.

Reply:
I'll leave this duplicated. 
a) it's easier to remove duplicate code than to unravel dependencies
b) the entire operator package is going away soon
c) it's the quickest path to merging this PR